docs/manager.md

```diff
@@ -24,20 +24,20 @@ The future owners of protocol on whose success the experiment of running a decen
 
 
 
-As the intialization function call can be seen above the manager is expected to provide certain paramters as:
+As the intialization function call can be seen above the manager is expected to provide certain paramters like:
 
 
 * Minimum amount
 
-    This refers to minimum investable amount that an investor can invest in the fund. The manager can set this according to their criteria of investment strategy to attract their niche level of investors.
+    This refers to the minimum investable amount that an investor is required to deposit in the fund. The manager can set this according to their criteria of investment strategy to attract their target audience.
 
 * Minimum return
 
-    The manager can set the minimum returns they will be targetting before collecting the performance fee on the profits made on the fund. The higher its set the better. This paramater once set cannot be changed and the contract will allow manager to collect there fee only when the minimum return criteria is met. The current range bound set is from 5-50% return on invesment
+    The manager can set the minimum returns they will be targetting before collecting the performance fee on the profitable returns made on the fund. The higher its set, the better in terms of gaining investors' confidence. This paramater once set cannot be changed and the contract will allow manager to collect their fee only when the minimum return criteria is met. The current value is set to be a minimum of 5% return on invesment.
 
 * Performance fee
 
-    The manager needs to set the performance fee they will be collecting when the minimum return set by them is met on total assets under management. The performance fee parameter is range bound and can be set in 5-50% range.
+    The manager needs to set the performance fee they will be collecting when the minimum return set by them is met on each investment independently. The performance fee parameter is range bound to a ceiling of 40% and 1/10th of the total fee collected by the Manager is allocated to the protocol.
```

Comment:
The manager needs to set the performance fee they will be collecting when the minimum return set by them is met on each individual investment. The performance fee parameter is range bound to a ceiling of 40% and 1/10th of the total fee collected by the Manager is allocated towards Investin.